### PR TITLE
git-delta: update completions

### PR DESCRIPTION
**Pull request by TomJo2000**
Update the completions for `delta` with the new ones generated by the upcoming `--generate-completion` option,
also adds the missing Fish and Zsh completions in the process.

### CODE REVIEW

**Comment by TomJo2000** [comment]
This is pretty much just a quality of life Revbump.

I'm also gonna go ahead and set myself as the maintainer for this package so I can be reminded to rip out the manual `curl` download once the `--generate-completion` option makes it into a stable release,
and instead do something more like `gh`.
https://github.com/termux/termux-packages/blob/17e6d9f172c491b4750d054fc419d9c20ba735a4/packages/gh/build.sh#L37-L44

I'd like to merge this in the morning if nobody has any objections.

**Review by truboxl — APPROVED** [review]
Approve until next release

Preferably use `termux_download` and `termux_step_post_get_source` instead of `curl` and `termux_step_post_make_install`

**Comment by TomJo2000** [comment]
Yep this was just a quick and dirty fix.
I just put it where the old completions hook was.

Let me quickly refactor that with `termux_download` and put it in `termux_step_post_get_source`

**Comment by TomJo2000** [comment]
Guess I should have done a *clean build* locally first, yep forgot to make the directory first, and `termux_download` doesn't.

**Comment by TomJo2000** [comment]
Sorry, decided to take a "2 hour nap" and it turned into 6.
Should be all good to merge.

**Pull request merged**
